Neo4j.Driver/Neo4j.Driver.Tests.TestBackend/SupportedFeatures.cs

```diff
@@ -63,8 +68,10 @@ static SupportedFeatures()
             "Optimization:EagerTransactionBegin",
             //"Optimization:ImplicitDefaultArguments",
             //"Optimization:MinimalResets",
-            "Optimization:PullPipelining"
+            "Optimization:PullPipelining",
+            "Optimization:AuthPipelining",
```

Comment:
Can we keep them alphabetical, just makes it a bit easier to search

Reply:
Done